tslint.json

```diff
@@ -111,7 +111,7 @@
   "rules": {
     // TypeScript Specific
     "member-access": [
-      true,
+      false,
```

Comment:
@victorschinaider I think it is important use this rule and mark properties and methods with accessors